src/test/resources/Login.feature

```diff
@@ -4,9 +4,20 @@ Feature: Login
   I want to sign in
   So I can use the app
 
-  @automated
+  Scenario Outline: User can not login with invalid value
+    Given login page is opened
+    When user inputs user name as "<username>" and password as "<password>"
+    And user clicks on login button
+    Then user sees error message as "<message>" is displayed
+    Examples:
+      | username  | password            | message                                                          |
+      |           |                     | Login failed! Please ensure the username and password are valid. |
+      | John Doe  |                     | Login failed! Please ensure the username and password are valid. |
+      |           | ThisIsNotAPassword  | Login failed! Please ensure the username and password are valid. |
+      | John Doe1 | ThisIsNotAPassword1 | Login failed! Please ensure the username and password are valid. |
+
   Scenario: User can sign in with valid value
-    Given user is in Katalon login page
+    Given login page is opened
     When user sets username as "John Doe" and password as "ThisIsNotAPassword"

```

Comment:
xóa step này ` user sets username as "John Doe" and password as "ThisIsNotAPassword"` và thay bằng step `user inputs user name as "<username>" and password as "<password>"`, xóa luôn cả trong stepDefine